src/api/routes/summaries.ts

```diff
@@ -138,15 +145,27 @@ async function findMissingCrudeSummariesforTmUser(tmUserId: string,
       await Promise.all(meeting.record_files.map(async file => {
         const transcriptId = file.record_file_id;
         const addrs = await getFileAddresses(file.record_file_id, tmUserId);
-
+        
         if (!await hasSummary(transcriptId, AI_MINUTES_SUMMARY_KEY) && 
           addrs.ai_minutes) {
+          const spkrList = await getSmartSpeakers(file.record_file_id, tmUserId);
+          spkrList.sort((a, b) => b.total_time - a.total_time);
+          let spkrInfo = "*发言时长统计（分钟)*: ";
+          spkrList.forEach((spkr, index) => {
+            spkrInfo += `${spkr.speaker_name}:${spkr.total_time}`;
+            if (index < spkrList.length - 1) {
+              spkrInfo += ", ";
+            }});
```

Comment:
don't format it here. create a function `formatSpeakerStats` and call it from `saveSummary`.